control/archiaus/panel.go

```diff
@@ -0,0 +1,54 @@
+package archiaus
+
+import (
```

Comment:
did you migrate archaius panel from go-chassis to mesher? then check it is the latest code or not